src/redux/reducers/samples/samplesError.js

```diff
@@ -8,6 +8,7 @@ const samplesError = (state, action) => {
     meta: {
       ...state.meta,
       saving: false,
+      loading: false,
```

Comment:
This was missing and in some cases when we had an error it would still show up in the ui as if we were loading forever